src/containers/message/sagas/sendMessage.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import {
+  call,
+  put,
+  takeEvery
+} from '@redux-saga/core/effects';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { SequenceAction } from 'redux-reqseq';
+
+import * as ChronicleApi from '../../../utils/api/ChronicleApi';
+import {
+  SEND_MESSAGE,
+  sendMessage,
+} from '../actions';
+
+const LOG = new Logger('MessageSagas');
+
+function* sendMessageWorker(action :SequenceAction) :Saga<*> {
+  try {
+    yield put(sendMessage.request(action.id));
+    const { messageDetailsList, organizationId } = action.value;
+
+    const response = yield call(ChronicleApi.sendMessage, organizationId, messageDetailsList);
+    if (response.error) throw response.error;
```

Comment:
This project is using axios directly which means `response.error` doesn't exist. This is what a response object looks like:

https://github.com/axios/axios/#response-schema